client/packages/inventory/src/Stocktake/api/operations.graphql

```diff
@@ -116,6 +116,26 @@ query stocktakeByNumber($stocktakeNumber: Int!, $storeId: String!) {
   }
 }
 
+fragment StockLineReducedBelowZeroError on StockLineReducedBelowZero {
```

Comment:
In this file I've added fragments, since errors would need to be extracted into error type it was easier to do it through framents:

https://github.com/openmsupply/open-msupply/blob/09a82f823dfddaf7f6537f49e8fccdf81280e88d/client/packages/inventory/src/Stocktake/context/stocktakeLineError.tsx#L10-L14

Otherwise typescript looks pretty complex (but can be done by looking at `UpsertStocktakeLinesMutation` type, but yeah it's very messy.